stdlib/arg.ml

```diff
@@ -304,20 +304,32 @@ let parse_expand l f msg =
 
 let second_word s =
   let len = String.length s in
-  let rec loop n =
-    if n >= len then len
-    else if s.[n] = ' ' then loop (n+1)
-    else n
+  let rec loop sep n =
+    if n >= len then sep, len
+    else if s.[n] = sep then loop sep (n+1)
+    else sep, n
   in
-  try loop (String.index s ' ')
-  with Not_found -> len
+  match String.index s '\t' with
+  | n -> loop '\t' n
```

Comment:
After looking at this a little more, even before this patch, any extra whitespace after the initial delimiter `' '` is actually displayed verbatim when displaying the `--help` listing.  That is, it cannot be used for source code alignment.

This being the case, I am changing the code so that only the first `\t` (if any) gets turned into a space and the rest gets included verbatim, as before.